bobtimus/src/lib.rs

```diff
@@ -258,10 +260,34 @@ where
         Ok(transaction)
     }
 
+    /// Handle Alice's loan offer request
+    ///
+    /// We return the possible loan terms that we offer to Alice
+    /// Alice can then request a loan that is within the parameters that were offered
+    pub async fn handle_loan_offer_request(&mut self) -> Result<LoanOffer> {
```

Comment:
At some point we should remove the name "Alice" (and "Bob") from bobtimus and replace it with "borrower" (and "lender" respectively).

Reply:
Yeah, good to keep it in mind, I started doing it now (i.e. where I touched code that contains Alice/Bob I change it where needed) but I did not clean up all instances. Might be a bit tricky - in a protocol context we will want to leave it I suppose.